ioctl/cmd/account/accountcreateadd.go

```diff
@@ -46,6 +46,8 @@ var accountCreateAddCmd = &cobra.Command{
 func init() {
 	accountCreateAddCmd.Flags().BoolVar(&CryptoSm2, "sm2", false,
 		config.TranslateInLang(flagSm2Usage, config.UILanguage))
+	accountCreateAddCmd.Flags().MarkHidden("sm2")
+
 }
```

Comment:
same here, CryptoSm2 won't be changed once command is compiled

Reply:
ok